packages/logger/src/Logger.ts

```diff
@@ -210,7 +213,7 @@ class Logger extends Utility implements ClassThatLogs {
    * @returns {void}
    */
   public addPersistentLogAttributes(attributes?: LogAttributes): void {
-    merge(this.persistentLogAttributes, attributes);
+    merge(this.persistentLogAttributes!, attributes);
```

Comment:
Why are we adding this assertion here?

Reply:
got a TS error during testing, but not during compile for some reason. Quickly added the assertion to remove the error, instead of debugging the `jest` / `ts-jest` config

Reply:
Just re-run tests, and the error is gone without the assertion. Clearly fixed my config after this. Will revert.